src/datumaro/plugins/data_formats/kaggle/base.py

```diff
@@ -79,19 +79,39 @@ def _parse_bbox_coords(self, bbox_str):
 
     def _load_annotations(self, datas: list, indices: Dict[str, int], bbox_flag: bool):
```

Comment:
Here, indices could be Dict[str, Dict[str, int]], too.

Reply:
Fix in [40ba7a4](https://github.com/openvinotoolkit/datumaro/pull/1607/commits/40ba7a4e8b43bd98d839b8ac259612bb51661237)